libs/ts4wing/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import { std, platform } from "@winglang/sdk";
+import { Construct } from "constructs";
+
+/**
+ * Properties for a Wing app.
+ */
+export interface WingAppProps {
```

Comment:
```suggestion
export interface AppProps {
```

There's already ambient semantic context, so no need to add "wing" here.